Header header
string command_id
string cart_id
string action
uint8 status
